content/en/docs/setup/additional-setup/getting-started/index.md

```diff
@@ -14,7 +14,7 @@ test: yes
 {{< tip >}}
 {{< boilerplate gateway-api-future >}}
 The following instructions allow you to get started with Istio using the Gateway API.
-If you prefer to use the tried-and-proven Istio classic API for traffic management, you should use
+If you prefer to use the tried-and-proven Istio APIs API for traffic management, you should use
```

Comment:
Redundant API? Seems weird saying the API API...
```suggestion
If you prefer to use the tried-and-proven Istio APIs for traffic management, you should use
```

Reply:
I missed it in my proofreading; good catch!